src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -459,7 +459,8 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            GregorianCalendar calendar = new GregorianCalendar();
```

Comment:
[CoreCLR](https://github.com/dotnet/coreclr/blob/fa5544c2b0589f1b1811dbfaf6f227327c4757fc/src/mscorlib/shared/System/Globalization/GregorianCalendar.cs#L87) already implemented this in `GregorianCalendar(...)`
We can use the implimentation.